config/initializers/doorkeeper.rb

```diff
@@ -149,7 +149,7 @@
   # `grant_type` - the grant type of the request (see Doorkeeper::OAuth)
   # `scopes` - the requested scopes (see Doorkeeper::OAuth::Scopes)
   #
-  # use_refresh_token
+  use_refresh_token
```

Comment:
I'm cool with this, just wanted to ask what prompted you to enable it

Reply:
I enabled it so that 3rd party wouldn't have to re-authorize. Figure it will smoothen the experience.